styles/main.css

```diff
@@ -0,0 +1,792 @@
+  }
+
+  .program-schedule {
+    width: 100%;
+    margin-top: 10px;
+  }
+
+  .program-schedule h4 {
+    font-size: 25px;
+  }
+
+  /* Main Programs Styles */
+  .main-programs {
+    width: 100%;
+    padding: 50px 20px;
+  }
+
+  .programs {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .program {
+    width: 100%;
+    margin: 10px 0;
+  }
+
+  /* Featured Tutors */
+  .featured-tutors {
+    width: 100%;
+    margin: 20px 0;
+    padding: 50px 10px;
+  }
+
+  .speaker-row {
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+  }
+
+  .featured-left,
+  .featured-right {
+    width: 94%;
+  }
+
+  .featured-left .image,
+  .featured-right .image {
+    width: 30%;
+  }
+
+  .image .image-top {
+    position: absolute;
+    top: 6px;
+    left: 8px;
+    width: 100%;
+  }
+
+  .speaker-info {
+    margin-left: 20px;
+    width: 100%;
+    padding: 5px 0;
+    padding-right: 20px;
+  }
+
+  /* Partners Section Styles */
+
+  .partner-logo-div {
+    width: 100%;
+    margin: 0 6px;
+  }
+
+  .partner-logo-div img {
+    width: 100%;
+  }
+
+  /* Footer Section Styles */
+
+  footer {
+    padding: 0 10px;
+  }
+
+  .footer-info h4 {
+    font-size: 18px;
+    line-height: 25px;
+    padding-top: 40px;
+    margin-left: 14px;
+  }
+
+  .footer-info {
+    padding: 0;
+  }
+
+  .footer-info p {
+    display: none;
+  }
+
+  .bootcapm-logo-div img {
+    width: 100%;
+  }
+
+  footer .footer-logo-image img {
+    width: 100%;
+  }
+
+  /* ABOUT PAGE MOBILE RESPONSIVE STYLES */
+
+  /* Hero Styles */
+  .about-hero {
+    height: calc(100vh - 90px);
+    background-image: url('../images/about/about-hero.svg');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+    padding: 100px 20px;
+  }
+
+  .about-line {
+    display: none;
+  }
+
+  .about-hero .hero-intro-text {
+    display: block;
+    text-align: center;
+    font-size: 18px;
+    font-weight: 400;
+    color: #ec5242;
+  }
+
+  .about-hero h1 {
+    font-size: 30px;
+  }
+
+  .about-hero-texts {
+    width: 96%;
+    padding: 20px 10px;
+    margin-top: 20px;
+    background-color: #fff;
+  }
+
+  .second-para,
+  .third-para {
+    display: none;
+  }
+
+  .contact-text {
+    width: 80%;
+    margin: auto;
+    margin-top: 20px;
+  }
+
+  .logo-description {
+    width: 80%;
+  }
+
+  .desktop-description {
+    display: none;
+  }
+
+  .logo-description .mobile-description {
+    display: block;
+    font-size: 18px;
+  }
+
+  /* Events Styles */
+  .events {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .events .event {
+    width: 90%;
+  }
+
+  /* Footer styles */
+  .about-footer {
+    width: 90%;
+    display: flex;
+  }
+
+  .about-footer div {
+    width: 50%;
+  }
+
+  .about-footer div img {
+    width: 100%;
+  }
+
+  /* CONTACT PAGE */
+  .contact-footer {
+    width: 100%;
+    padding: 40px 20px;
+  }
+
+  .footer-devbootcamp-logo {
+    width: 100%;
+  }
+}
```

Comment:
**Changes Required**
- [ ] Good effort on using the media query. However, the approach you have used is referred to as `Desktop-first Design`. For this project, we want to use the `Mobile-First Design` approach. we start out designing at the smallest viewport possible, which we will call a **mobile viewport**, and we then progressively add styles and sometimes other enhancements as the viewport increases using media queries.

Reply:
Ok, I'll see to that. I just find it time-consuming to use that approach first. But will get used to it anyways.
I don't know if one may call it a habit or what, but I'm so much used to designing for desktop first and then applying styles for Mobile. Since all end up being responsive, I just think it's the same thing.
👍

Reply:
- [ ] Mobile-First approach is not just about being responsive, responsiveness is just one of the many benefits. Please check out [this article](https://highlandsolutions.com/blog/why-mobile-first-design-is-important) on what exactly Mobile-First approach is.

Reply:
Well solved